apps/aecore/lib/aecore/channel/worker.ex

```diff
@@ -0,0 +1,509 @@
+defmodule Aecore.Channel.Worker do
```

Comment:
as I understand this is one GenServer instance to handle state of all channels of this node. Wouldn't it be better to spawn a separate process for each channel? This way others would life if one dies.

Reply:
I think it's something to consider, but the channels states are going to be stored on this so even if it dies everything should get reloaded. (and it should handle everything safely, so it never dies)

Reply:
okay, lets keep it in mind as possible follow up task